tests/autoyast/clone.pm

```diff
@@ -19,11 +19,17 @@
 use strict;
 use base 'console_yasttest';
 use testapi;
+use version_utils 'is_sle';
 
 sub run {
     my $self = shift;
     assert_script_run 'rm -f /root/autoinst.xml';
-    assert_script_run 'yast2 --ncurses clone_system', 400;
+    script_run('(yast2 --ncurses clone_system; echo yast2-clone_system-status-\$?) | tee /dev/$serialdev', 0);
+    if (is_sle('15+') && check_screen 'autoyast2-install-accept', 30) {
```

Comment:
Please use a multi-tag `assert_screen` with `match_has_tag` instead of `check_screen` with non-zero timeout to prevent introducing any timing dependant behaviour, to save test execution time as well as state more explicitly from the testers point of view what are the expected alternatives. For example:

```perl
assert_screen([qw(yast2_console-finished yast2_missing_package)]);
if (match_has_tag('yast2_missing_package')) {
    send_key 'alt-o';  # confirm package installation
    assert_screen 'yast2_console-finished';
}
```

Also I don't think this should be regarded as really "SLE15" specific behavior as just depending on the previous step and the selected packages for installation the packages might be missing. I guess with the above proposed multi-tag assert_screen we can use that without any further product version checks in all cases without any time lost.

Reply:
Thanks. No problem, I updated code following your suggestion and remove the product version checks also.
Verify log: http://openqa-apac1.suse.de/tests/3068#